stores/nft.ts

```diff
@@ -24,14 +30,36 @@ export const useNFTStore = defineStore('nft', () => {
     return iscnStore.getISCNRecordDataByIdPrefix(iscnIdPrefix)
   })
 
-  function addNFTClass(nftClass: NFTClass) {
-    nftClassByIdMap.value[nftClass.id] = nftClass
+  function addLegacyNFTClass(nftClass: LegacyNFTClass) {
+    legacyNFTClassByIdMap.value[nftClass.id] = nftClass
   }
 
-  function addNFTClasses(nftClasses: NFTClass[]) {
+  function addLegacyNFTClasses(nftClasses: LegacyNFTClass[]) {
     const nftClassIds: string[] = []
     for (const nftClass of nftClasses) {
```

Comment:
Not sure if it is better to just use `addNFTClass` and detect `isEVM` inside `stores/nft`, that way user don't need to know if they are using Legacy or EVM NFT store.
Is splitting these action/getter is meaningful for components? We can keep the state splitted

Reply:
I tried to use the same method to support different types of NFTClass, but it turned out to be a TypeScript nightmare